README.md

```diff
@@ -6,6 +6,56 @@
 
 # SW Helpers
 
+## Contents
+
+- [Ways to Use the Libraries](#ways-to-use-the-libraries)
+- [Setting Up](#setting-up)
```

Comment:
Three forms generally used in titles IT documents:
* Noun ("Setup")
* Gerund ("Setting Up")
* Infinitive ("To Set Up")
What we try for is consistency within a single doc set. I'll see what we do most in this set and make them consistent.